src/lib/web3/connectors.ts

```diff
@@ -5,16 +5,17 @@ import { FortmaticConnector } from '@web3-react/fortmatic-connector';
 import { PortisConnector } from '@web3-react/portis-connector';
 import { NetworkConnector } from '@web3-react/network-connector';
 
-import { Chains } from './index';
 import { config } from '../../config';
 
+const chainId = Number(config.supportedChainId);
```

Comment:
not a big deal, so don't bother changing it right now, but this should just be typed at this point. instead of making it an id/number, it should just be a `Chain`

Reply:
ok, i'll change it